### PR TITLE
Zulu 2020-04-14 update

### PR DESCRIPTION
**Pull request by jorsol**
7.0.262-zulu
13.0.3-zulu
14.0.1-zulu

### CODE REVIEW

**Comment by jorsol** [comment]
Comments or issues with this PR?

File: src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala [hunk 1]
```diff
@@ -257,5 +257,97 @@ class AzulZuluMigrations {
...
+        "java",
+        "7.0.262-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu7.38.0.11-ca-jdk7.0.262-win_x64.zip",
+        Windows,
+        Some(Zulu)
+      ),
+      Version(
+        "java",
+        "7.0.262-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu7.38.0.11-ca-jdk7.0.262-macosx_x64.zip",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala:280]: We only support zips for Windows at this time. Could you change this to be a tarball please?

File: src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala [hunk 2]
```diff
@@ -257,5 +257,97 @@ class AzulZuluMigrations {
...
+        "java",
+        "13.0.3-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu13.31.11-ca-jdk13.0.3-win_x64.zip",
+        Windows,
+        Some(Zulu)
+      ),
+      Version(
+        "java",
+        "13.0.3-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu13.31.11-ca-jdk13.0.3-macosx_x64.zip",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala:311]: As above.

File: src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala [hunk 3]
```diff
@@ -257,5 +257,97 @@ class AzulZuluMigrations {
...
+        "java",
+        "14.0.1-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu14.28.21-ca-jdk14.0.1-win_x64.zip",
+        Windows,
+        Some(Zulu)
+      ),
+      Version(
+        "java",
+        "14.0.1-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu14.28.21-ca-jdk14.0.1-macosx_x64.zip",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala:342]: As above.

**Comment by jorsol** [comment]
Done, changed binary format of mac to tgz.

**Comment by marc0der** [comment]
Awesome, thanks for contributing!

**Pull request merged**